.github/workflows/run-tests.yml

```diff
@@ -58,16 +58,16 @@ jobs:
             patch-qt-3d: false
             LD_PRELOAD: /lib/x86_64-linux-gnu/libSegFault.so
 
-          # - distro-version: 'rawhide'
-          #   qt-version: 6
-          #   run-tests: false
-          #   docker-tag-suffix: ''
-          #   with-qt6: ON
-          #   with-qt5: OFF
-          #   with-3d: OFF
-          #   with-quick: OFF
-          #   patch-qt-3d: false
-          #   LD_PRELOAD: ''
+          - distro-version: 'rawhide'
```

Comment:
@3nids , will this need to be updated to 34 too?

Reply:
not strictly needed, right now (as it's not yet a parameter in the Dockerfile), let's wait for the PR to be green